docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
Have you run ``docker-compose run web tox -e docs`` to see if the Sphinx docs still build after this change?